### ISSUE TITLE
Support filelist paths relative to the PWD

### ISSUE THREAD

**Issue by keith** [nicklockwood/SwiftFormat#1285]
Currently if I create a temporary file to use as the `--filelist` containing:

```
Modules/AccidentReportingServerDrivenService/Sources/Canvas/Elements/FilePicker.swift
```

my format invocation fails with:

```
error: File not found at /var/folders/gj/wf3swl0x215b2sq1qy84kzkm0000gn/T/Modules/AccidentReportingServerDrivenService/Sources/Canvas/Elements/FilePicker.swift.
```

Personally I would expect this to be relative to the PWD similar to swiftc / clang vs relative to the filelist file

**Comment by nicklockwood** [comment]
This would be a breaking change so it's unlikely to happen unless the feature is currently unusable. I envisioned that people would typically place a manually-created the filelist in a project-relative location, like the .swiftformat config file.

If it's created temporarily as part of a build step then presumably it's not a problem to just use absolute paths?

**Comment by keith** [comment]
Makes sense. Transforming to absolute paths is fine just a bit of extra work, in our case I'm generating the temporary file list from `git diff --name-only` to operate only on changes

**Comment by nicklockwood** [comment]
Thinking about it some more, I could probably check both locations and apply whichever one matched. There might be some cases where it's ambiguous if you had wildcard matches, but that's unlikely.
